crates/cdk-mint-rpc/build.rs

```diff
@@ -0,0 +1,4 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    tonic_build::compile_protos("src/proto/cdk-mint-rpc.proto")?;
```

Comment:
```suggestion
    println!("cargo:rerun-if-changed=src/proto/cdk-mint-rpc.proto");
    tonic_build::compile_protos("src/proto/cdk-mint-rpc.proto")?;
```

Helps avoid unnecessary recompilations. With this, `cargo` will only recompile if the proto changed.

Reply:
Nice thank you